mobile-widgets/qmlmanager.cpp

```diff
@@ -315,7 +345,7 @@ void QMLManager::finishSetup()
 		int error = parse_file(existing_filename);
 		if (error) {
 			// we got an error loading the local file
-			appendTextToLog(QString("got error %2 when parsing file %1").arg(existing_filename, get_error_string()));
+			appendTextToLog(QString("error parsing file %1").arg(existing_filename));
```

Comment:
it would append the same message `parse_file` previous appends via the callback. i think the above call should be removed.